### DISCUSSION TITLE
Achievement

### DISCUSSION THREAD

**Discussion by thewalkerz** [Heroic-Games-Launcher/HeroicGamesLauncher#2728] [Q&A] [answered]
Is Heroic for Windows and Linux compatible with Epic and GoG achievements?

**Comment by imLinguin** [comment, marked as answer]
With Epic yes, if you enable EOS Overlay for the game.  
For GOG we are currently in a process of testing our custom solution https://www.gamingonlinux.com/2023/05/comet-is-an-open-source-implementation-of-gog-galaxy-sdk-that-needs-testing/

**Reply by novhack** [reply]
What is the status of integrating Comet into Heroic?

**Reply by imLinguin** [reply]
Comet itself is still incomplete. I'm working towards complete offline support too

**Reply by adosix** [reply]
Hi, can we get current update on getting achievements from GOG  work on heroic launcher for linux?

according this issue:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1279 
the implementation will be merged to the heroic, can you elaborate please and give approximation of time window.

**Reply by imLinguin** [reply]
Hi, [comet](https://github.com/imLinguin/comet) is almost done, it lacks some logic related to leaderboards and a few debug calls.

I generally hate giving ETAs but for this I'd expect full Heroic integration in the upcoming months, with more launchers to follow later on

**Reply by adosix** [reply]
Hi. care to share some updates if there are any? :) (sorry for bothering you)

**Reply by imLinguin** [reply]
2.15.0 ships comet as experimental feature, make sure it's enabled in advanced settings

**Comment by SuperSamus** [comment]
Technically, achievements on the Epic Games Store are supported out of the box with Legendary, but you get no notifications.
